README.md

```
@@ -0,0 +1,6 @@
+# Uniswap V4 SDK Rust
+
+[![Rust CI](https://github.com/shuhuiluo/uniswap-v4-sdk-rs/actions/workflows/rust.yml/badge.svg)](https://github.com/shuhuiluo/uniswap-v4-sdk-rs/actions/workflows/rust.yml)
+
+A Rust SDK for building applications on top of Uniswap V4. Rewrite of the
+TypeScript [V4 SDK](https://github.com/Uniswap/sdks).
```

Markdownlint

Files should end with a single newline character

(MD047, single-trailing-newline)